lib/configuration-adapter.js

```diff
@@ -162,46 +162,95 @@ function ConfigurationAdapter(socketAddress, sessionToken)
     return requestPromise.get(options);
   }
 
 
+  /**
+   * Set an entity in the ViSense system.
+   *
+   * @param {string} name The query entity name.
+   * @param {string} type The query entity type.
+   * @param {string} value The new value of the query entity.
+   *
+   * @returns {object} A promise with the query result. The resolve/reject callback prototypes
+   *                    are: - resolve({string} query result)
+   *                         - reject({string} error message)
+   */
+  function set(name, type, value)
+  {
+    let options = JSON.parse(JSON.stringify(_defaultOptions));
+    options.url += '?command=set&' + name + '=' + value + '&filter=' + type;
+    return requestPromise.get(options);
+  }
+
+
   // Class prototype
   let ConfigurationAdapterProto =
     {
       /**
-       * Query for a parameter.
+       * Get a parameter.
+       *
+       * @param {string} name The parameter name.
+       *
+       * @returns {object} A promise with the query. The resolve/reject callback prototypes
+       *                    are: - resolve({string} parameter value)
+       *                         - reject({string} error message)
+       */
+      getParameter(name)
+      {
+        return get(name, 'parameter').then(extractParameterValue);
+      },
+
+
+      /**
+       * Get an event signal.
+       *
+       * @param {string} name The signal name.
+       *
+       * @returns {object} A promise with the query. The resolve/reject callback prototypes
+       *                    are: - resolve({string} signal state)
+       *                         - reject({string} error message)
+       */
+      getSignal(name)
+      {
+        return get(name, 'signal').then(extractSignalState);
+      },
+
+
+      /**
+       * Set a parameter.
        *
        * @param {string} name The parameter name.
+       * @param {string} value The new value of the parameter.
        *
        * @returns {object} A promise with the query. The resolve/reject callback prototypes
        *                    are: - resolve({string} parameter value)
        *                         - reject({string} error message)
        */
-      queryParameter(name)
+      setParameter(name, value)
       {
-        return query(name, 'parameter').then(extractParameterValue);
+        return set(name, 'parameter', value);
       },
 
 
       /**
-       * Query for an event signal.
+       * Set an event signal.
        *
        * @param {string} name The signal name.
+       * @param {string} value The new value of the signal.
        *
        * @returns {object} A promise with the query. The resolve/reject callback prototypes
        *                    are: - resolve({string} signal state)
        *                         - reject({string} error message)
        */
-      querySignal(name)
+      setSignal(name, value)
       {
-        return query(name, 'signal').then(extractSignalState);
+        return set(name, 'signal', value);
```

Comment:
This is not possible. One is only able to trigger an event slot by setting a `slot` type input. That's why I would also like to recommend a function name change to `setSlot`.